engine/src/flutter/lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -50,6 +50,8 @@ class SurfaceVertices implements ui.Vertices {
     return list;
   }
 
+  bool get isEmpty => positions.isEmpty;
```

Comment:
nit: There could be several empty things: positions, colors, indices. Just `isEmpty` doesn't seem specific enough?

Reply:
Renamed to `hasNoPoints` for clarity